Buildings/Media/Specialized/Water/HighTemperature.mo

```diff
@@ -0,0 +1,118 @@
+within Buildings.Media.Specialized.Water;
+package HighTemperature
+  "Package with model for liquid water with constant density and an extended 
+  maxiumum temperature threshold"
+  extends IBPSA.Media.Water(
+    T_max=273.15+600);
+  replaceable function saturationTemperature
+    "Return saturation temperature from a given pressure"
+    extends Modelica.Icons.Function;
+    input AbsolutePressure p
+      "Pressure";
+    output Temperature T
+      "Saturation temperature";
+  protected
+    Real a[:]={2.2830066E+02,1.1893913E+00,5.2484699E-01,1.2416857E-01,-1.3714779E-02,5.5702047E-04}
+      "Coefficients";
+    Real logP=log(p)
+      "Log of pressure";
+  algorithm
+    T := a[1]+a[2]*logP+a[3]*logP^2+a[4]*logP^3+a[5]*logP^4+a[6]*logP^5
+      "Saturation temperature";
+    annotation (
+      smoothOrder=2,
+      Documentation(
+        info="<html>
+    <p>
+    Saturation temperature is computed from pressure. This relation is
+    valid in the region of <i>273.16</i> to <i>647.096</i> K (<i>613.3</i> to <i>22,049,100</i> Pa).
+    </p>
+    <p>
+    The function has the following form:
+    </p>
+    <p align=\"center\" style=\"font-style:italic;\">
+    T = a<sub>1</sub> + a<sub>2</sub> ln(p) + a<sub>3</sub> ln(p)<sup>2</sup> +
+    a<sub>4</sub> ln(p)<sup>3</sup> + a<sub>5</sub> ln(p)<sup>4</sup> + a<sub>6</sub> ln(p)<sup>5</sup>
+    </p>
+    <p>
+    where temperature <i>T</i> is in units Kelvin, pressure <i>p</i> is in units Pa, and <i>a<sub>1</sub></i>
+    through <i>a<sub>6</sub></i> are regression coefficients.
+    </p>
+  </html>",
+        revisions="<html>
+<ul>
+<li>
+March 6, 2020, by Kathryn Hinkelman:<br/>
+First implementation.
+</li>
+</ul>
+</html>"));
+  end saturationTemperature;
+  annotation (
```

Comment:
Should this model have a defaultComponentName?

Reply:
Per recent conversations I had with Michael, this model shouldn't be necessary. I will delete it all together.